clustergroup/templates/subscriptions.yaml

```diff
@@ -15,6 +15,16 @@ spec:
   sourceNamespace: {{ default "openshift-marketplace" $subs.sourceNamespace }}
   channel: {{ default "stable" $subs.channel }}
   installPlanApproval: {{ coalesce $installPlanValue $.Values.global.options.installPlanApproval }}
+  {{- if $subs.config }}
+  {{- if $subs.config.env }}
+  config:
+    env:
+  {{- range $subs.config.env }}
+      - name: "{{ .name }}"
```

Comment:
Why the extra quotes here? Just curious as it's the only place in the file where we use them

Reply:
I took the above quotes out.